conn.go

```diff
@@ -37,6 +37,10 @@ type Conn struct {
 	locker    sync.Mutex
 }
 
+var (
+	AuthRequiredErr = fmt.Errorf("Please authenticate first.")
```

Comment:
The Go convention is to name error values `ErrXXX`, so the name should be `ErrAuthRequired`.

(`Backend.AnonymousLogin` doc comment should be updated too)

Reply:
Changed in e198df1